### ISSUE TITLE
application not responding on Pixel 8 and 8 Pro devices with busy url content

### ISSUE THREAD

**Issue by ndbeyer** [react-native-webview/react-native-webview#3351]
**Bug description:**

On android Pixel 8 and 8 Pro devices (Android 14) we had "application not responding" crashes when react-native-webview@13.8.1 loaded a url where quite a lot of stuff got rendered. 
- This did not happen when using the link via Chrome on the same devices
-  It did not happen on any other devices also not on a Samsung S24 with also Android 14 installed on it
- On devices where the webview did not crash, adb logcat reported: `Choreographer(16451): Skipped 82 frames! The application may be doing too much work on its main thread` [described here](https://medium.com/@banerjee.s.sayans/skipped-611-frames-the-application-maybe-doing-too-much-work-on-its-main-thread-69d86c171a56), so I thought this might be the underlying problem
- Crashlytics showed entries related to
  -   libmonochrome_64
  -  libwebviewchromium_plat_support
  -  libhwui.so 
  -  libc.so 
  -  libutils.so 
- setting `<Wrapper enderToHardwareTextureAndroid={true}><WebView/><Wrapper/>` did not help

### TLDR: setting `<WebView androidLayerType="software"  />` instead of `<WebView  />` fixed the problem

**To Reproduce:**
reproduction was only seen on urls that I cannot post here.

**Expected behavior:**
WebView should not crash on only specific devices.

**Environment:**
 - OS: Android
 - OS version: 14
 - react-native version: 0.72.6
 - react-native-webview version: 13.8.1

**Comment by andidev** [comment]
@ndbeyer having the same issue actually. 
Not sure its the webview causing it but would guess that since most of the application is in the webview for us.
`androidLayerType="software"` makes app really slow so that is not an option.

@ndbeyer Did you manage to solve this somehow? Getting really critical for us here since we cannot downgrade since lower api levels cannot be released anymore in play store.

**Comment by ndbeyer** [comment]
@andidev setting `androidLayerType="software"` was sufficient for the use case. Did not look into it any further, so I cannot provide any further advice.
